terraform-modules/aws/amplify/README.md

```diff
@@ -0,0 +1,27 @@
+# AWS Amplify App Terraform Module
```

Comment:
`terraform-doc` should be ran on this module to create the docs.

Reply:
output will be like this: https://github.com/ManagedKube/kubernetes-ops/pull/428/files#diff-dfaa5dde6b8ced55169546ad5282f14812d3fcd7282c3ab2fd3a33cbb9af18e3R1

Reply:
updated